src/Query/Grammar.php

```diff
@@ -189,7 +189,8 @@ protected function buildSearchFunction(string $function, array $where): string
     }
 
     /**
-     * @inheritDoc
+     * {@inheritDoc}
+     * @return false
```

Comment:
Why tho?

Reply:
I wasn't aware of it (I changed it because I could show the shape more narrowly), but it was unnecessary. I will change it back.